example/web/package.json

```diff
@@ -1,14 +1,15 @@
 {
-    "name": "pesapaljs-exemple",
-    "version": "0.0.2",
-    "scripts":{
-      "start":"node app.js"
-    },
-    "dependencies":{
-        "express":"4.x.x",
-        "morgan":"1.5.x",
-        "body-parser":"x.x.x",
-        "jade":"x.x.x",
-        "lorem-ipsum": "x.x.x"
-    }
+  "name": "pesapaljs-exemple",
+  "version": "0.0.2",
+  "scripts": {
+    "start": "node app.js"
+  },
+  "dependencies": {
+    "body-parser": "x.x.x",
+    "express": "4.x.x",
+    "jade": "x.x.x",
+    "lorem-ipsum": "x.x.x",
+    "morgan": "1.5.x",
+    "oauthsimple": "0.0.1"
```

Comment:
`oauthsimple` is already a dependency of the library. if `pesapaljs`'s dependencies are installed, we don't need this.